MyHome.UI/RecurringIncomeInput.cs

```diff
@@ -232,6 +238,8 @@ private void MultiDaySave()
             }
         }
 
+
+
         private void CreateNewIncome(DateTime dtCurrentSaveDate)
         {
             Income newIncome = new Income(decimal.Parse(txtAmount.Text), dtCurrentSaveDate, Convert.ToInt32(cmbCategory.SelectedValue), Convert.ToInt32(cmbPayment.SelectedValue), txtDetail.Text);
```

Comment:
This method does not actually save the income.
Add a call like the following:
`_incomeService.Create(newIncome)`